skiko/src/awtMain/cpp/windows/directXRedrawer.cc

```diff
@@ -21,10 +21,37 @@
 #include <dxgi1_6.h>
 
 #include <dcomp.h>
-#pragma comment(lib, "dcomp.lib")
 
 const int BuffersCount = 2;
 
+namespace DCompLibrary {
+    static bool isInit = false;
```

Comment:
No need to separate init flag. just use something like
```cpp
static HINSTANCE getDCompositionLibrary() {
    static HINSTANCE library = LoadLibrary("dcomp.dll");
    return library;
}
```

Reply:
Done